chapter_deep-learning-computation/read-write.md

```diff
@@ -65,7 +65,11 @@ x2
 
 ```{.python .input}
 #@tab tensorflow
-x2 = np.load("x-file.npy", allow_pickle=True)
+# Note that the out_type here is the type of the serialized
+# tensor that we wrote to a file previously. It must match the
+# type of the serialized tensor and no implicit conversion
+# will take place. 
+x2 = tf.io.parse_tensor(tf.io.read_file("x-file"), out_type=tf.int32)
```

Comment:
how about save a list? we still use np right now. also this APIs are not quite clean. maybe just the preview np.load/np.save approach